client/src/Components/FeelingsLog/FeelingsLog.js

```diff
@@ -21,7 +21,7 @@ const FeelingsLog = ({logs}) => {
         <td>{log.date.toLocaleString(DateTime.DATE_MED)}</td>
         <td>{toTitleCase(log.feeling)}</td>
         <td>{log.activity}</td>
-        <td>{<Link to={`/how-you-felt/entry/${log.entry_id}`}>📓</Link>}</td>
+        <td>{<Link to={`/how-you-felt/entry/${log.entry_id}`}><i class="fa-solid fa-book"></i></Link>}</td>
```

Comment:
I'm curious why you used an i tag here! isn't this the arrow?

Reply:
it is! good question. This is the html auto generated by Font Awesome, so I have no idea.